src/controller/python/test/test_scripts/mobile-device-test.py

```diff
@@ -151,18 +150,58 @@ def main():
               "Failed to test Read Basic Attributes")
 
     logger.info("Testing subscription")
-    FailIfNot(test.TestSubscription(nodeid=1, endpoint=LIGHTING_ENDPOINT_ID),
+    FailIfNot(test.TestSubscription(nodeid=device_nodeid, endpoint=LIGHTING_ENDPOINT_ID),
               "Failed to subscribe attributes.")
 
     logger.info("Testing another subscription that kills previous subscriptions")
-    FailIfNot(test.TestSubscription(nodeid=1, endpoint=LIGHTING_ENDPOINT_ID),
+    FailIfNot(test.TestSubscription(nodeid=device_nodeid, endpoint=LIGHTING_ENDPOINT_ID),
               "Failed to subscribe attributes.")
 
     logger.info("Testing on off cluster over resolved connection")
-    FailIfNot(test.TestOnOffCluster(nodeid=1,
+    FailIfNot(test.TestOnOffCluster(nodeid=device_nodeid,
                                     endpoint=LIGHTING_ENDPOINT_ID,
                                     group=GROUP_ID), "Failed to test on off cluster")
 
+
+def do_tests(controller_nodeid, device_nodeid, timeout, discriminator, setup_pin, commissioning, log_level):
+    timeoutTicker = TestTimeout(timeout)
+    timeoutTicker.start()
+
+    test = BaseTestHelper(nodeid=controller_nodeid)
+
+    coloredlogs.install(level='DEBUG')
+    chip.logging.RedirectToPythonLogging()
+
+    # logging.getLogger().setLevel(logging.DEBUG)
+    logging.getLogger().setLevel(log_level)
+
+    commissioning_method = None
+
+    if commissioning == 'eth':
```

Comment:
Until we actually have an alternative commissioning modality, we should just omit this argument.

Reply:
Removed